include/openenclave/internal/eeid.h

```diff
@@ -22,16 +24,23 @@ static const uint8_t OE_DEBUG_PUBLIC_KEY[] = {
     0xce, 0x73, 0xe4, 0x33, 0x63, 0x83, 0x77, 0xf1, 0x79, 0xab, 0x44,
     0x56, 0xb2, 0xfe, 0x23, 0x71, 0x93, 0x19, 0x3a, 0x8d, 0xa};
 
+/** Layout of the EEID memory page. */
+typedef struct
+{
+    OE_SHA256 config_id;
```

Comment:
Mainly because `oe_eeid_t` was a dirty hack :-) The config_id has now moved into `oe_eeid_t`, but the user now submits a `oe_enclave_setting_eeid_t`, which contains only the bits they should care about.